SmartAIAutocomplete/SmartAIAutocomplete/AIService/AzureOpenAIService.cs

```diff
@@ -64,7 +64,7 @@ public AzureOpenAIService()
 
         private void ValidateCredential()
         {
-            if (client == null)
+            if (client == null && IsCredentialValid)
```

Comment:
@Kamalesh-Periyasamy why we need IsCredentialValid in here?

Reply:
An exception was thrown if we ran the sample without credentials, so if we add this, that line was skipped and moved to next for credential validation sir.